packages/fast-components-foundation-react/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "moduleResolution": "node",
+        "module": "ES6",
+        "target": "ES6",
+        "baseUrl": ".",
+        "declaration": true,
+        "jsx": "react",
+        "outDir": "./dist"
```

Comment:
does the outDir actually get used?  I've noticed on some occasions this gets copied over from other projects but not all the attributes are used.  Let's remove any that are not needed.

Reply:
This is used anywhere we are exporting packages for distribution. This will be used here based on the setup